### ISSUE TITLE
Confirming match issue made me banned 

### ISSUE THREAD

**Issue by Xlexandr** [ValveSoftware/Dota2-Gameplay#6567]
### Description

Im unable to confirm match, it says at the bottom of the screen : Confirming match... 

and i get ban if i close program as its frozen state

i waited for match confirmation for like 10 min

**Comment by denkata07** [comment]
+1, got banned 3 times and is still looping between finding and confirming match.

**Comment by Xlexandr** [comment]
ok, i found that if i cancel match at "Finding" stage even after "Confirming" it not bans me if i quit

**Comment by denkata07** [comment]
Yeah its a normal behavior. However if its stuck at confirming for 10
minutes, it gets annoying and furthermore - ban.

**Comment by woocash88** [comment]
SAme

**Comment by Squirrel2k9** [comment]
I have similar issue, now I have banned for next 1h. @Valve How posssible is to update servers without close any interactions on it?

**Comment by DSDV** [comment]
same here ... wantet to get some of the rerolls which are only available in the last 3.5 hours of the battlepass .... 

guess no luck there ...

**Comment by maritz** [comment]
Seems like this is the same as #6383 and thus maybe not related to what I thought was VPN issues.

Has happened to me yesterday (jan 11th) twice again.

**Comment by Fonzzz** [comment]
How to fix this shit guys?

**Comment by mahdyyahoo** [comment]
So many gauntlets gone to waste...

**Comment by Avarise831** [comment]
only eu west server problem

**Comment by ravkishore27** [comment]
upvote

**Comment by ChrisLauinger77** [comment]
Same here today 1 time 2 minutes penalty next time 30

**Comment by Su33ke** [comment]
valve just dont care

**Comment by EdwinMan** [comment]
is anyone still having this issue?

**Comment by DSDV** [comment]
@EdwinMan hard to replicate now that the servers aren't that overloaded. But my guess is it still exists. 
You just don't have the required load on the match making orchestrator anymore (or at least only on rare times)

**Comment by Yoblipa** [comment]
+1. im having same issue, stick at confirming till i shut down and banned twice now

**Comment by Yoblipa** [comment]
> only eu west server problem

i had the issue in argentina server too

**Comment by Yoblipa** [comment]
> is anyone still having this issue?

me, today

**Comment by EdwinMan** [comment]
yea I am still having it in US West server :(

**Comment by marcelsby** [comment]
I'm having this problem here in São Paulo server. I've got 15 min of ban.

**Comment by OriEy** [comment]
Also have this from time to time. Really annoying.

**Comment by gbrb1** [comment]
Happening to me since yesterday, any fixes?
